### ISSUE TITLE
Access denied, you need (at least one of) the SUPER privilege(s) for this operation

### ISSUE THREAD

**Issue by shinesoftware** [markshust/docker-magento#501]
Hello Mark,

thanks for your job! It is very useful! I get this error when I try to import my existing database.

```
$ bin/clinotty mysql -hdb -umagento -pmagento magento < src/backup.sql 
ERROR 1227 (42000) at line 390: Access denied; you need (at least one of) the SUPER privilege(s) for this operation
```
how have I to solve it?

thanks

**Comment by shinesoftware** [comment]
I have solved all by myself in this way!

`sed 's/\sDEFINER=`[^`]*`@`[^`]*`//g' -i src/backup.sql`

**Comment by markshust** [comment]
That's a neat one-liner!

Another option is to use the root MySQL user:

```
bin/clinotty mysql -hdb -uroot -pmagento magento < src/backup.sql 
```

**Comment by markshust** [comment]
I'm going to re-open this, so we can add this to the documentation. This is quite common, and I hear this come up more than I should 😄 .

**Comment by Stoyvo** [comment]
> That's a neat one-liner!
> 
> Another option is to use the root MySQL user:
> 
> ```
> bin/clinotty mysql -hdb -uroot -pmagento magento < src/backup.sql 
> ```

This absolutely needs to be in the README. Thank you for sharing @markshust

**Comment by markshust** [comment]
Sorry @Stoyvo didn't see your commit until I just pushed mine up. I added this info as another blurb, containing both of these lines. I think that usually MySQL commands should not be ran as root, probably for non-sensical reasons, but having that option there right in the readme is definitely helpful. I would probably prefer removing the DEFINER lines and continuing to import the backup with the standard MySQL user -- should probably look into this to find out which is more ideal.
